README.md

```diff
@@ -77,6 +77,14 @@ Alternatively, you can use a regex:
 plug CORSPlug, origin: ~r/https?.*example\d?\.com$/
 ```
 
+And also you can put configuration into config.exs:
+```elixir
+config :cors_plug,
+  origin: ["foo.bar"],
+  max_age: 86400,
+  methods: ["GET", "POST"]
+```
```

Comment:
Please specificy what overwrites what (currently app config overwrites defaults, options passed to the plug override app config).